examples/diffusers/worker.py

```
@@ -1,76 +1,80 @@
+DEFAULT_SIZE = "1280x720"
+DEFAULT_SECONDS = 5
+DEFAULT_FPS = 24
+DEFAULT_NUM_FRAMES = 125
+DEFAULT_NUM_INFERENCE_STEPS = 50
+DEFAULT_GUIDANCE_SCALE = 1.0
+DEFAULT_SEED = 1024
+DEFAULT_MAX_VIDEO_WIDTH = 4096
+DEFAULT_MAX_VIDEO_HEIGHT = 4096
+DEFAULT_RESPONSE_FORMAT = "b64_json"
+DEFAULT_OUTPUT_FORMAT = "mp4"
+DEFAULT_OUTPUT_DIR = "/tmp/dynamo-fastvideo-outputs"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/dynamo-fastvideo-outputs"

(S108)